packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -534,6 +534,46 @@ void main() {
         equals('TextInputAction.done'));
   });
 
+  testWidgets('connection is closed when TextInputClient.onConnectionClosed message received', (WidgetTester tester) async {
+    Future<void> verifyVisibility(bool expectedVisibility) async {
```

Comment:
Why is this method async and returning a Future?

Reply:
Also, what do we benefit from actually defining this method? The test seems clearer to my by just putting the expect directly where this method is called?

Reply:
Sure, I'll delete it.

There was another one like this in the same file, so I thought it is common practice to define a verify method first.

Reply:
done.